File: 250719/100점 도달하기/reach-100-point.java
// import
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // 입력
        Scanner sc = new Scanner(System.in);
        int intScore = sc.nextInt();

        // 반복 및 조건문
        for (int i = intScore; i < 101; i++) {
            if (i >= 90) {
                System.out.print("A ");
            } else if (i >= 80) {
                System.out.print("B ");
            } else if (i >= 70) {
                System.out.print("C ");
            } else if (i >= 60) {
                System.out.print("D ");
            } else {
                System.out.print("F ");
            }
        }
    }
}

File: 250719/100점 도달하기/reach-100-point.py
# 입력
intScore = int(input())

# 반복 및 조건문
for i in range(intScore, 101, 1) :
    if i >= 90 :
        print("A", end = " ")
    elif i >= 80 :
        print("B", end = " ")
    elif i >= 70 :
        print("C", end = " ")
    elif i >= 60 :
        print("D", end = " ")
    else :
        print("F", end = " ")